fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -237,6 +238,14 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 				traceBatch = { TraceBatch{} };
 				traceBatch.get().addEvent("GetValueDebug", a.debugID.get().first(), "Reader.Before");
 			}
+			if (now() - a.startTime > SERVER_KNOBS->ROCKSDB_READ_VALUE_TIMEOUT) {
+				TraceEvent(SevError, "RocksDBError")
```

Comment:
I have mixed feelings about whether this should be an error or a warning. I'll let others put in their $0.02.